pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -2110,3 +2057,47 @@ func GetVolumeNameByTarget(domain *api.Domain, target string) string {
 	}
 	return ""
 }
+
+func DetectEFIConfig(arch, ovmfPath string, secureBoot bool) (*EFIConfiguration, error) {
+	secureLoader := "yes"
+	efiCode := EFICode
+	efiVars := EFIVars
+
+	if !secureBoot {
+		secureLoader = "no"
+	}
+
+	if secureBoot {
+		efiCode = EFICodeSecureBoot
+		efiVars = EFIVarsSecureBoot
+	} else if isARM64(arch) {
+		efiCode = EFICodeAARCH64
+		efiVars = EFIVarsAARCH64
+	}
+
+	_, err := os.Stat(filepath.Join(ovmfPath, efiCode))
+	if os.IsNotExist(err) && efiCode == EFICode {
```

Comment:
I think this function has a little bit an incomplete error handling. I think that any error which is not `IsNotExist` will make the function pass. Like if the file-system can't be accessed, the function would just pass. Is that the case?

Reply:
Ouch you're right, and the error was present in the original PR as well, thanks for spotting this, I'll re-push shortly.